crates/rspack/tests/tree-shaking/class-extend/expected/main.js

```diff
@@ -0,0 +1,37 @@
+(self['webpackChunkwebpack'] = self['webpackChunkwebpack'] || []).push([["main"], {
+"./app.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+'use strict';
+__webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  'v': function() { return v; }
+});
+/* harmony import */var _lib__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./lib */"./lib.js");
+
+ class Lib extends /* "./lib" unused */null {
```

Comment:
The webpack used `null` instead of `undefined`, because it can be class extends.